src/app/index/index.component.css

```diff
@@ -4,5 +4,23 @@
 
 #search-bar{
   position: absolute;
-  left: 300px;
+}
+
+#set-susper-default{
+  display:none;
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
@sandeepm96 make a separate pull for the responsive design .dont push into this pull